cli/src/bin/phylum.rs

```diff
@@ -173,15 +142,6 @@ async fn handle_commands() -> CommandResult {
         .await;
     }
 
-    let mut api = PhylumApi::new(
-        &mut config.auth_info,
-        &config.connection.uri,
-        timeout,
-        ignore_certs,
-    )
-    .await
-    .context("Error creating client")?;
-
     // PhylumApi may have had to log in, updating the auth info so we should save the config
     save_config(&config_path, &config).with_context(|| {
```

Comment:
As noted in the comment, `save_config` needs to happen after `PhylumApi::new()` in order to save new credentials if a login occurred.

Reply:
We should then probably factor all of this behavior out into a separate `async fn`, in order to not have the logic split over many different points.